runtime/queries/html/textobjects.scm

```diff
@@ -0,0 +1,23 @@
+(element
+  (start_tag)
+  (_)* @xml_element.inside
+  (end_tag)?
+. ) @xml_element.around
```

Comment:
Looks like there's some odd interaction here with the `*` vs the `?` actually. I think we may need to split these into two separate stanzas so that the version with the `end_tag` takes precedence:

```suggestion
(element
  (start_tag)
  (_)* @xml_element.inside
  (end_tag)
. ) @xml_element.around

(element
  (start_tag)
  (_)* @xml_element.inside
. ) @xml_element.around
```

Otherwise a snippet like:

```html
    <a href="print.html" title="Print this book" aria-label="Print this book">
        <i id="print-button" class="fa fa-print"></i>
    </a>
```

If the cursor is on the whitespace at the start of line 2, `mie` will select the `</a>`